### DISCUSSION TITLE
override block-tune-delete

### DISCUSSION THREAD

**Discussion by rogueturnip** [codex-team/editor.js#2306] [Ideas]
I'm trying to make it so a block can't be deleted and was wondering if it's possible to override the delete tune button?

Also, I was playing with setting "tune" in the config and if I add "delete" in the array it changes the order of them in the tune menu.  Any chance I'm missing a setup there that could suppress the default tune actions?

It seems that there are a lot of questions about preventing delete so it seems that the tune array or having the block tune as an external plugin might be ways to address this.

**Comment by rogueturnip** [comment]
I was playing around with this and it is possible to override the default one by creating a Tune tool that does the same a delete.  The only problem is not having access to any details about the block that would allow disabling the delete based on the configuration of the block.

**Reply by isage** [reply]
Can you please elaborate on "creating a Tune tool that does the same a delete"?
